src/runtime/initialize-component.ts

```diff
@@ -32,7 +32,7 @@ export const initializeComponent = async (
     // Let the runtime know that the component has been initialized
     hostRef.$flags$ |= HOST_FLAGS.hasInitializedComponent;
 
-    if (BUILD.lazyLoad || BUILD.hydrateClientSide) {
+    if (BUILD.lazyLoad) {
```

Comment:
We feel a bit unsure about removing `BUILD.hydrateClientSide` from the if statement as we can't tell what side effects that has. We suggest to achieve the same result by looking up the `lazyBundleId` of `cmpData` which guarantees that we only enter the if statement in the case of transpiling a lazy loaded component:
```suggestion
    const bundleId = cmpMeta.$lazyBundleId$;
    if ((BUILD.lazyLoad || BUILD.hydrateClientSide) && bundleId) {
```

Reply:
My feeling is that this was originally an unintendedly introduced condition - but just to be sure I implemented your suggestion, to make sure we don't introduce breaking changes.